server/routes/stock-details.js

```diff
@@ -6,6 +6,33 @@ const defaultData = {
     stock_details: {}
 }
 
```

Comment:
bring in some useful realtime data too, i.e. current price, change today, etc. can do in future commit, just a heads up :)